spec/confluent_spec.lua

```diff
@@ -53,15 +53,18 @@ describe("Testing Confluent Cloud", function()
     assert.is_not_nil(brokers)
     assert.is_same("b0-" .. confluent_bootstrap_server_adr, brokers[0].host)
     assert.are.equal(brokers[0].port, 9092)
+
+    local first_partition = partitions[0]
+    table.sort(first_partition.replicas)
+    table.sort(first_partition.isr)
```

Comment:
sort replicas and in-sync replicas to make sure they pass the below assertion when comparing with `same`